Gemfile

```diff
@@ -58,8 +58,13 @@ group :development do
 end
 
 group :test do
+  gem 'rails-controller-testing'
   gem "webdrivers", "~> 4.4"
 end
 
 # do not rely on host’s timezone data, which can be inconsistent
 gem "tzinfo-data"
+
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+
+gem "govuk_design_system_formbuilder", "~> 2.1"
```

Comment:
I wonder if the team has strong opinions on when to use double or single quotes. DfE probably don't mind either otherwise they would have set up a rule in rubocop

Reply:
I have a rubocop rule enforcing double quotes. Not sure whether that came from DfE or elsewhere though

Reply:
Ah, here https://github.com/alphagov/rubocop-govuk/blob/master/config/style.yml#L52